code/game/atoms.dm

```diff
@@ -1769,3 +1769,27 @@
 /atom/proc/InitializeAIController()
 	if(ai_controller)
 		ai_controller = new ai_controller(src)
+
+/**
+ * Point at an atom
+ *
+ * Moved from pointed() verb in mob.dm
+ *
+ * Intended to enable and standardise the pointing animation for all atoms
+ *
+ * Not intended as a replacement for the mob verb
+ */
+/atom/proc/pointat(atom/A as mob|obj|turf in view())
```

Comment:
I don't think this sort of signature makes sense in a proc?

Reply:
I didn't think about this because I just copypasted the code like I'm Terry B. Davis but it worked so here we are